web/workflow.R

```diff
@@ -215,4 +215,4 @@ if (PEcAn.utils::status.check("FINISHED") == 0) {
 }
 
 db.print.connections()
-print("---------- PEcAn Workflow Complete ----------")
+print("---------- PEcAn Workflow Complete ----------")
```

Comment:
workflow.R hasn't been updated for the new functions and function arguments, and thus will fail to run. This is probably why your SIPNET test is failing.